plugins/woocommerce-admin/client/order-attribution/use-order-attribution-install-banner.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import {
+	OPTIONS_STORE_NAME,
+	PLUGINS_STORE_NAME,
+	useUser,
+} from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+
+const OPTION_NAME_BANNER_DISMISSED =
+	'woocommerce_order_attribution_install_banner_dismissed';
+const OPTION_VALUE_YES = 'yes';
+
+const useOrderAttributionInstallBanner = () => {
```

Comment:
Can we document this hook more? Like how it works and why it is for.

Reply:
Added in 5c89dee